include/bout/mesh.hxx

```diff
@@ -562,7 +565,35 @@ class Mesh {
    * Set the parallel (y) transform from the options file
    */
   void setParallelTransform();
+
+    //Region related routines
+
+  /// Get the named region from the region_map for the data iterator
+  ///
+  /// Throws if region_name not found
+  Region<> &getRegion(const std::string &region_name){
+    return getRegion3D(region_name);
+  }
+  Region<ind3D> &getRegion3D(const std::string &region_name);
+  Region<ind2D> &getRegion2D(const std::string &region_name);
   
+  /// Add a new region to the region_map for the data iterator
+  ///
+  /// Outputs an error message if region_name already exists
+  void addRegion(const std::string &region_name, Region<> region){
+    return addRegion3D(region_name, region);
```

Comment:
why return in void function?

Reply:
In case we decide that these routines should actually return something/to demonstrate that these two routines have the same return type. Does this harm anything?